### PR TITLE
Fixes #13072: support-info script doesn't correctly detect jetty process with Rudder 4.3+

### PR DESCRIPTION
**Pull request by el-k0022**
https://issues.rudder.io/issues/13072

### CODE REVIEW

**Comment by el-k0022** [comment]
Commit modified

File: rudder-agent/SOURCES/absolute-aix-path [hunk 1]
```diff
@@ -29,7 +29,7 @@ relink() {
   if [ "${mode}" = "bin" ]; then
     files=$(for i in ${base}/*; do file $i; done | grep "executable" | sed 's/:.*//')
   elif [ "${mode}" = "so" ]; then
-    files=$(for i in ${base}/*.so.*; do file $i; done | grep "executable" | sed 's/:.*//')
+    files=$(for i in ${base}/*.so*; do file $i; done | grep "executable" | sed 's/:.*//')
```
Comment (peckpeck) [inline rudder-agent/SOURCES/absolute-aix-path]: What is thod doing here ?

**Comment by el-k0022** [comment]
Commit modified

**Comment by el-k0022** [comment]
Commit modified

**Comment by el-k0022** [comment]
Commit modified

**Comment by peckpeck** [comment]
You stille have a useless commit here

**Pull request closed without merging**